src/index.html

```diff
@@ -1,16 +1,32 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Search Form Example</title>
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="search-form search-form_big" data-qa="big">
+      <label class="search-form__label search-form__label_big">
+        <input type="text" 
+          class="search-form__input search-form__input_big" 
+          placeholder="Try &quot;Los Angeles&quot;" 
+          data-qa="keypress"
+        >
```

Comment:
```suggestion
        <input 
          type="text" 
          class="search-form__input search-form__input_big" 
          placeholder="Try &quot;Los Angeles&quot;" 
          data-qa="keypress"
        >
```
Follow the correct code style.
You can find the rules here: https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md

<img width="993" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/102293862/3249723f-ad8a-408b-845c-79f9f0bae288">